### PR TITLE
feat(telescope): add grep_string keymap

### PR DESCRIPTION
**Pull request by Groveer**

### CODE REVIEW

File: lua/keymap/init.lua [hunk 1]
```diff
@@ -231,6 +231,7 @@ local plug_map = {
 		:with_silent()
 		:with_desc("editn: Change current direrctory by zoxide"),
 	["n|<leader>fb"] = map_cu("Telescope buffers"):with_noremap():with_silent():with_desc("find: Buffer opened"),
+	["n|<leader>fs"] = map_cu("Telescope grep_string"):with_noremap():with_silent():with_desc("find: Current word"),
```
Comment (aarnphm) [inline lua/keymap/init.lua:234]: qq: would live_grep_args achive the same?

Comment (Groveer) [reply lua/keymap/init.lua:234]: no, live_grep_args must input args, this command-line will grep the string of the current cursor

Comment (Groveer) [reply lua/keymap/init.lua:234]: It will help me to find strings quickly

Comment (Jint-lzxy) [reply lua/keymap/init.lua:234]: @aarnphm `grep_string` is equivalent to passing the string under cursor as the first parameter to `live_grep_args`. User input inside that prompt is equivalent to appending additional parameters to `live_grep_args`.

**Review by Jint-lzxy — APPROVED** [review]
LGTM 🎐

File: lua/keymap/init.lua (hunk 1 above)
Comment (aarnphm) [reply lua/keymap/init.lua:234]: ah got it.

**Pull request merged**